src/table/utils/handle-key-press.ts

```diff
@@ -54,35 +54,42 @@ export const getNextCellCoord = (
   evt: React.KeyboardEvent,
   rootElement: HTMLElement,
   cellCoord: [number, number],
-  topAllowedRow = 0
+  allowedRows: {
+    selected: {
+      top: number;
+      bottom: number;
+    };
+    top: number;
+    bottom: number;
+  } = { selected: { top: 0, bottom: 0 }, top: 2, bottom: 1 }
 ): [number, number] => {
   const rowCount = rootElement.getElementsByClassName('sn-table-row').length;
   const columnCount = rootElement.getElementsByClassName('sn-table-head-cell').length;
   let [nextRow, nextCol] = cellCoord;
 
   switch (evt.key) {
     case 'ArrowDown':
-      nextRow < rowCount - 1 && nextRow++;
+      nextRow < rowCount - 1 - allowedRows.selected.bottom && nextRow++;
       break;
     case 'ArrowUp':
-      nextRow > topAllowedRow && nextRow--;
+      nextRow > allowedRows.selected.top && nextRow--;
       break;
     case 'ArrowRight':
-      // topAllowedRow greater than 0 means we are in selection mode
-      if (topAllowedRow > 0) break;
+      // allowedRows.top greater than 0 means we are in selection mode
+      if (allowedRows.selected.top > 0) break;
       if (nextCol < columnCount - 1) {
         nextCol++;
-      } else if (nextRow < rowCount - 1) {
+      } else if (nextRow < rowCount - 1 - allowedRows.bottom) {
```

Comment:
I was wrong here. In selection mode you can not go left or right at all, so no need to change anything here :)

Reply:
well...not totally. Out of selection mode arrow used to move the cell to Totals row, it is not how the old table works. Now with the new condition it works fine and  Totals row is excluded. I fixed it for Top case as well.

Reply:
Sorry I don't understand. On row 79 we make sure that ArrowRight does not do anything when in selection mode. When not in selection mode, you should be able to reach totals, both with up and down, and left and right

Reply:
The left and right arrow was working correctly before this fix, so no need to change anything in the right and left case

Reply:
Umm... ok. I was not aware of it. Just for being sure, in old table Totals row is not reachable by left/right arrow in both selection and normal mode, just up/down works, in the current table all works as moving to another cell except in selection mode, correct?

Reply:
yes. if you only press right it goes through the cells like a book: left to right, then to to the start of the row below. The only thing you need to fix in this PR is the condition for the down arrow

Reply:
Ok, I'll revert the change then.